app/pages/index.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import Link from 'next/link';
+
+const Title = styled.h1`
+  font-size: 50px;
+  color: ${({ theme }) => theme.colors.primary};
+`;
+
+export default function Home() {
+  return (
+    <>
+      <Link href="/1">
+        &#9997; Create Profile
+      </Link>
```

Comment:
Can we have the link as [a button](https://button-inc.github.io/service-development-toolkit/@button-inc/bcgov-theme/index.html?path=/docs/components-button--default)?

Since this page is where a user will be presented with information about the application they're about to complete maybe we could get a bit of [lorem ipsum](https://www.lipsum.com/) to fill the page so we can see the call to action button in context?

Reply:
Yeah that sounds good. I was also thinking it needed something more!